docs/overview/architecture-overview.md

```diff
@@ -52,9 +52,12 @@ chamber with its own users, channels, customizations, and so on. This
 means that one person might be a user of multiple Zulip realms. The
 administrators of an organization have a great deal of control over
 who can register an account, what permissions new users have, etc. For
-more on security considerations and options, see [the security model
-section](../production/security-model.md) and the [Zulip Help
-Center](https://zulip.com/help/).
+more on security considerations and options, see our [guide on securing
+your Zulip server](../production/security-model.md) and the [Zulip help
+center](https://zulip.com/help/).
```

Comment:
Should this link to zulip.com/security here too?

Reply:
Oops, I half-finished that change.